Automattic/sensei
# WPML: Fix course outline for a course translation

Resolves #7392 

In this PR we address an issue with a course translation.
The easiest way to observe the issue is to create a translation for a course with lessons and check the Course Outline in the translated course.

To fix it, we need:
- Update `_lesson_course` meta field in translated (duplicated) lessons to point the new course.
- Update 'module' taxonomies for translated lessons.
- Create lesson translations (duplicates) if they don't exist yet. 

## Proposed Changes
* Fix course outline for a course translation.
* Create a lesson duplicates when a course translation is created.

## Testing Instructions

1. Create a course.
2. Add a few modules with lessons, a standalone lesson.
3. Create translations for lessons.
4. Create a translation for the course.
5. Make sure the course translation has an outline with translated lessons.
6. Now create a new course with lessons.
7. Create a translation for the course.
8. Go to the translated course and make sure there are lessons in the Course Outline.

## Pre-Merge Checklist
